lib/ares/http.rb

```diff
@@ -1,13 +1,14 @@
 module Ares
   module Http
     def get(url, params = {})
-      response = HTTParty.get(url, query: params)
+      require 'pry'; binding.pry
```

Comment:
Remove debugger entry point `binding.pry`.<br>Do not use semicolons to terminate expressions.